components/ai_chat/resources/page/components/uploaded_img_item/style.module.scss

```diff
@@ -39,35 +40,25 @@
   align-items: flex-start;
 }
 
-.nameAndExtensionRow {
-  display: flex;
-  flex-direction: row;
-  justify-content: flex-start;
-  align-items: center;
-}
-
 .forEllipsis {
   display: grid;
 }
 
 .nameText {
+  cursor: pointer;
   font: var(--leo-font-default-regular);
   color: var(--leo-color-text-primary);
   white-space: nowrap;
   overflow: hidden;
   text-overflow: ellipsis;
 }
 
-.extensionText {
-  font: var(--leo-font-default-regular);
-  color: var(--leo-color-text-primary);
-}
-
 .sizeText {
   font: var(--leo-font-small-regular);
   color: var(--leo-color-text-tertiary);
 }
 
 .removeButton {
   flex-grow: 0;
+  --leo-icon-color: var(--leo-color-icon-secondary);
```

Comment:
nit: variables should be the first lines in a style

Reply:
Moved above!